Module/main.js

```diff
@@ -199,12 +199,19 @@ class BugReportForm extends FormApplication {
    */
   get dependencies() {
     return this.module.data.dependencies?.map((dependency) => {
-      const mod = game.modules.get(dependency.name);
+      let plugin = game.modules.get(dependency.name);
+      if (plugin === undefined){
+        if (dependency.name === game.system.data.name) {
+          plugin = game.system;
+        } else if (dependency.name === game.world.data.name) {
+          plugin = game.world;
+        }
+      }
```

Comment:
I believe there's a `type` field on dependency which could be used in a `switch` here:
```js
switch (dependency.type) {
  case "module":
// ...
  case "system":
// ...
  case "world":
// ...
}
```